app/controllers/bulk_downloads_controller.rb

```diff
@@ -22,6 +22,9 @@ def types
       end
     end
 
+    # Filter out types that should not be part of the creation UI because they are created offline by admins
+    download_types = download_types.reject { |type| type[:execution_type] == MANUAL_EXECUTION_TYPE }
```

Comment:
I would add a new property to the types in `BulkDownloadTypesHelper` called `hide_in_creation_modal` and setting `hide_in_creation_modal: true` for CUSTOMER_SUPPORT_BULK_DOWNLOAD_TYPE, instead of attaching this filtering behavior to the MANUAL_EXECUTION_TYPE execution type.

Ideally, each property should have a single clear purpose.

Reply:
Great point, I've made that change